corehq/apps/events/tasks.py

```diff
@@ -47,7 +48,7 @@ def get_existing_cases_by_user_ids(domain: str) -> dict[str, CommCareCase]:
 
 
 # TODO: Test
-def get_case_block_for_user(user: CommCareUser, owner_id: str) -> CaseBlock:
+def get_case_block_for_user(user: CommCareUser, owner_id: str, attendee_case_type) -> CaseBlock:
```

Comment:
Alternatively
```suggestion
def get_case_block_for_user(user: CommCareUser, owner_id: str) -> CaseBlock:
    attendee_case_type = get_attendee_case_type(user.domain)
```

---
We will, of course, drop all these useful type hints that allow us to see the properties of `user` immediately. :smirk:  But in a different universe, if we were to keep them, we would apply [this style guide](https://docs.google.com/document/d/1q90SaGuxV9dKnjnEIDdu8ma7xrZJxJq_HsfkCUhQqqk/edit?pli=1#heading=h.78jkbnn4l08n), and give `attendee_case_type` a type hint too:
```suggestion
def get_case_block_for_user(
    user: CommCareUser,
    owner_id: str,
    attendee_case_type: str,
) -> CaseBlock:
```